.github/workflows/ci.yml

```
@@ -166,19 +167,328 @@ jobs:
-    needs: [typecheck, e2e-plan]
+      && github.event.pull_request.head.repo.full_name == github.repository
+    needs: [typecheck, e2e-plan, e2e-lane-plan, e2e-release-build]
     runs-on: ubuntu-latest
     timeout-minutes: 90
+    strategy:
+      fail-fast: false
+      max-parallel: 4
+      matrix: ${{ fromJSON(needs.e2e-lane-plan.outputs.matrix) }}
+    concurrency:
+      group: release-qualification-e2e-${{ matrix.lane }}
+      queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)